### DISCUSSION TITLE
livewire form submit button not working on ios devices

### DISCUSSION THREAD

**Discussion by Temian1** [livewire/livewire#5166] [1. Help]
so created a withdraw form andriod users where able to request withdraw , but iphone users cannot place withdraw , i confirmed that when they click on submit button nothing happens

```
   <form wire:submit="withdraw({{ $user->id }})" class="">
        <!--<input wire:model="amount" type="text" placeholder="withdrawal amount" class="w-full h-12 rounded-md">-->
        <input wire:model="amount" type="text" placeholder="withdrawal amount"  class="input input-bordered input-accent w-full max-w-xs"  @if($confirm == 1)  disabled @endif  />
      @if ($user->total_balance >= 4500)
       @if($confirm !== 1)
          <button type="submit" class="bg-blue-500 rounded-md px-3 py-2 text-white mt-4">request withdraw</button>
          @else
          <button class="bg-yellow-500 rounded-md px-3 py-2 text-white mt-4">pending withdraw</button>
       @endif
   

   @else
   <button disabled class="bg-red-500 rounded-md px-3 py-2 text-white mt-4">Not Eligible For Withdraw</button>
   @endif
     </form>

**Comment by Martleby** [comment]
I don't know if this will help, but might be worth trying:

```blade
<form wire:submit.prevent="
```

**Comment by joshhanley** [comment]
@Temian1 yeah you should be using `.prevent` with `wire:submit` to stop the browser trying to submit the form, as @Martleby suggested.

You should also be using `wire:key` on any element inside conditionals/loops to ensure that it gets morphed correctly when Livewire updates. See here for more details https://laravel-livewire.com/docs/2.x/troubleshooting#dom-diffing-issues

So you need to add `wire:key` to your buttons.

Hope this helps!
